Source/SpeechToTextV1/SpeechToTextAudioStreamer.swift

```diff
@@ -20,11 +20,12 @@ import AVFoundation
 /** Stream microphone audio to Speech to Text. */
 class SpeechToTextAudioStreamer: NSObject, AVCaptureAudioDataOutputSampleBufferDelegate {
```

Comment:
Yup. The `AVCaptureAudioDataOutputSampleBufferDelegate` protocol requires the class to inherit from `NSObject`.

If I remove inheritance from `NSObject`, the project fails to build because of the following error: `Type 'SpeechToTextAudioStreamer' does not conform to protocol 'NSObjectProtocol'.`